ui/lib/pki/addon/templates/configuration/index.hbs

```diff
@@ -0,0 +1,39 @@
+<SecretListHeader
+  @model={{this.model}}
+  @backendCrumb={{hash
+    label=this.model.id
+    text=this.model.id
+    path="vault.cluster.secrets.backend.list-root"
+    model=this.model.id
+  }}
+  @isEngine={{true}}
+/>
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @params={{array "configuration.tidy"}}>
+      Tidy
+    </ToolbarLink>
+    <ToolbarLink @params={{array "configuration.edit"}}>
+      Edit configuration
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+
+<div class="form-section">
+  <label class="title has-padding-top is-5">
+    URLs
+  </label>
+</div>
+<div class="form-section">
+  <label class="title has-padding-top is-5">
+    CRL
+  </label>
+</div>
+<div class="form-section">
+  <label class="title has-padding-top is-5">
+    Mount Configuration
+  </label>
+  <div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+    {{! ARG TODO component and empty state }}
+  </div>
+</div>
```

Comment:
I know this is still WIP but wanted to confirm that this will become a component because I think it's used in other routes as well?

Reply:
Good catch. The empty state section will, yep, and then I'll use the infoTableRows components for the three chunks of data dispaled.